Source/control.cpp

```diff
@@ -1777,13 +1716,13 @@ void DrawDurIcon(CelOutputBuffer out)
 			return;
 	}
 
-	x = PANEL_X + PANEL_WIDTH - 32 - 16;
+	int x = PANEL_X + PANEL_WIDTH - 32 - 16;
 	if (!hasRoomUnderPanels) {
 		if (invflag || sbookflag)
 			x -= SPANEL_WIDTH - (gnScreenWidth - PANEL_WIDTH) / 2;
 	}
 
-	p = &plr[myplr];
+	PlayerStruct *p = &plr[myplr];

```

Comment:
Don't see a reason why this is a pointer inster of a reference?
The review won't let me mark the entire block for some reason, but I think this should be preferred:

const PlayerStruct &p = plr[myplr];
x = DrawDurIcon4Item(out, p.InvBody[INVLOC_HEAD], x, 4);
x = DrawDurIcon4Item(out, p.InvBody[INVLOC_CHEST], x, 3);
etc...

Reply:
why?

Reply:
It doesn't need to be reseated, it can't ever be a nullptr that you would want to handle, and all access is read only. All of that speaks for making it a const&.

Reply:
reseated as in you cannot reassign it to a different instance during the lifetime of the function?

Reply:
I like how this sounds, but let's keep it for later as an actual refactor. Who knows maybe there is even a way to configure it in clang-tidy.

Reply:
Yeah. There's a bit of a base rule of "Make const& if possible, make * only if necessary".
Plus, typing "." is shorter than "->" :P